Unable to build OS X target w. Carthage

I am just starting to transition from pods to carthage, so it is highly possible I am doing something wrong :smile: 

Running `carthage update` fails with the following error while building the os x target:

```
The following build commands failed:
	CompileSwift normal x86_64 /Users/dev/Documents/Projects/swift/PGW-OSX/Carthage/Checkouts/AlamofireObjectMapper/AlamofireObjectMapperTests/AlamofireObjectMapperTests.swift
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(2 failures)
ld: warning: directory not found for option '-F/Users/dev/Documents/Projects/swift/PGW-OSX/Carthage/Checkouts/AlamofireObjectMapper/Carthage/Checkouts/Alamofire/build/Debug-iphoneos'
ld: warning: directory not found for option '-F/Users/dev/Documents/Projects/swift/PGW-OSX/Carthage/Checkouts/AlamofireObjectMapper/Carthage/Checkouts/ObjectMapper/build/Debug-iphoneos'
ld: warning: directory not found for option '-F/Users/dev/Library/Developer/Xcode/DerivedData/AlamofireObjectMapper-dauotuqewdtvbqbcptlcgtwkqndr/Build/Products/Debug-iphoneos'
ld: warning: directory not found for option '-F/Users/dev/Documents/Projects/swift/PGW-OSX/Carthage/Checkouts/AlamofireObjectMapper/Carthage/Checkouts/Alamofire/build/Debug-iphoneos'
ld: warning: directory not found for option '-F/Users/dev/Documents/Projects/swift/PGW-OSX/Carthage/Checkouts/AlamofireObjectMapper/Carthage/Checkouts/ObjectMapper/build/Debug-iphoneos'
ld: warning: directory not found for option '-F/Users/dev/Library/Developer/Xcode/DerivedData/AlamofireObjectMapper-dauotuqewdtvbqbcptlcgtwkqndr/Build/Products/Debug-iphoneos'
ld: warning: directory not found for option '-F/Users/dev/Documents/Projects/swift/PGW-OSX/Carthage/Checkouts/AlamofireObjectMapper/Carthage/Checkouts/Alamofire/build/Debug-iphoneos'
ld: warning: directory not found for option '-F/Users/dev/Documents/Projects/swift/PGW-OSX/Carthage/Checkouts/AlamofireObjectMapper/Carthage/Checkouts/ObjectMapper/build/Debug-iphoneos'
ld: warning: directory not found for option '-F/Users/dev/Library/Developer/Xcode/DerivedData/AlamofireObjectMapper-dauotuqewdtvbqbcptlcgtwkqndr/Build/Products/Debug-iphoneos'
ld: warning: directory not found for option '-F/Users/dev/Documents/Projects/swift/PGW-OSX/Carthage/Checkouts/AlamofireObjectMapper/Carthage/Checkouts/Alamofire/build/Debug-iphoneos'
ld: warning: directory not found for option '-F/Users/dev/Documents/Projects/swift/PGW-OSX/Carthage/Checkouts/AlamofireObjectMapper/Carthage/Checkouts/ObjectMapper/build/Debug-iphoneos'
ld: warning: directory not found for option '-F/Users/dev/Library/Developer/Xcode/DerivedData/AlamofireObjectMapper-dauotuqewdtvbqbcptlcgtwkqndr/Build/Products/Debug-iphoneos'
/Users/dev/Documents/Projects/swift/PGW-OSX/Carthage/Checkouts/AlamofireObjectMapper/AlamofireObjectMapperTests/AlamofireObjectMapperTests.swift:11:8: error: no such module 'ObjectMapper'
A shell task failed with exit code 65
```

My `Carthage` file is bare-bones:

```
github "tristanhimmelman/AlamofireObjectMapper" ~> 0.7
```

I am getting the same `no such module 'ObjectMapper'` error in the IDE when I clone the `AlamofireObjectMapper` repo, run `carthage update`, open the workspace and then set the target to `AlamofireObjectMapper OS X`. Hitting build makes the error disappear, and I am able to run the tests afterwards. 

I am running XCode 6.4 (6E35b)

Any ideas?

--- a/AlamofireObjectMapper.xcodeproj/xcshareddata/xcschemes/AlamofireObjectMapper OSX.xcscheme
+++ b/AlamofireObjectMapper.xcodeproj/xcshareddata/xcschemes/AlamofireObjectMapper OSX.xcscheme
@@ -22,10 +22,10 @@
          </BuildActionEntry>
          <BuildActionEntry
             buildForTesting = "YES"
-            buildForRunning = "YES"
+            buildForRunning = "NO"
             buildForProfiling = "NO"
             buildForArchiving = "NO"
-            buildForAnalyzing = "YES">
+            buildForAnalyzing = "NO">
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "6AB2A08E1AF27DF1001EBB20"

